src/common/widgetIcons/Video.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function Video({ styling }) {
```

Comment:
@sathya716 Would it be possible for us to reuse the Video Icon that we already have? Or alternatively would it be appropriate for us to replace it everywhere with this one? (an example of the current one is on the Media page in the Dashboard) 
`src/common/icons/Video.js`